src/wallet/wallet.cpp

```diff
@@ -227,10 +227,10 @@ static std::set<std::string> g_loading_wallet_set GUARDED_BY(g_loading_wallet_mu
 static std::set<std::string> g_unloading_wallet_set GUARDED_BY(g_wallet_release_mutex);
 
 // Custom deleter for shared_ptr<CWallet>.
-static void ReleaseWallet(CWallet* wallet)
+static void FlushAndDeleteWallet(CWallet* wallet)
 {
     const std::string name = wallet->GetName();
-    wallet->WalletLogPrintf("Releasing wallet\n");
+    wallet->WalletLogPrintf("Releasing wallet %s..\n", name);
```

Comment:
I didn't think much about it. Just made https://github.com/bitcoin/bitcoin/pull/30659#discussion_r1718356981 happy while was tackling https://github.com/bitcoin/bitcoin/pull/30659#discussion_r1718396246. Either way is fine for me.

Reply:
Done. Reverted.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30659#discussion_r1718510823

> Can you explain why a four-line diff requires a five-line scripted diff? Seems easier to just have a normal diff when the script is longer than the diff.

Agree that a manual change is fine here, but IMO unless a scripted diff is badly written or fragile (e.g. hardcoding line numbers), I'd always prefer it over a manual change because it:

1. Documents and summarizes changes in a representation verified by CI.
2. Helps with rebasing and cherrypicking. I often paste scripted diffs commands manually when I have a conflict to resolve, and have code which runs scripted-diffs automatically when cherrypicking.
3. Helps verify changes are complete. For example in rename commits, can verify there are not code comments referring to the old name.
4. Can prevent silent conflicts because of the checking done by CI.